shopify-client/yarn.lock

```diff
@@ -913,6 +913,15 @@
   resolved "https://registry.yarnpkg.com/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz#2b5a3ab3f918cca48a8c754c08168e3f03eba61b"
   integrity sha512-shAmDyaQC4H92APFoIaVDHCx5bStIocgvbwQyxPRrbUY20V1EYTbSDchWbuwlMG3V17cprZhA6+78JfB+3DTPw==
 
+"@nuxt/opencollective@^0.3.2":
```

Comment:
Do other devs need to worry about these dependencies?

Reply:
If no one has worked the frontend just pull this PR and yarn install. You should have everything you need.